database_wrapper.py

```diff
@@ -34,6 +34,52 @@
 ]
 UNION_PROPERTIES = Union[ProfessorsProperties]
 
+EXPECTED_KEYS_BY_ENTITY = {
+    AudioSampleMetaData : [
+        "isWakeWord",
+        "firstName",
+        "lastName",
+        "gender",
+        "noiseLevel",
+        "location",
+        "tone",
+        "timestamp",
+        "username",
+        "filename",
+    ],
```

Comment:
Possibly not required since the AudioSampleMetaData is being saved in its own special function

Reply:
This is good